modules/tasks.js

```diff
@@ -0,0 +1,164 @@
+const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+function saveTasks() {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+export function renderTasks() {
```

Comment:
Great job with this function, also here you can use arrow function and improve the efficiency of your code,